### ISSUE TITLE
adding a .gitignore file

### ISSUE THREAD

**Issue by surchs** [brainhack-school2020/emilyemchen-EEGML#4]
Hey, great repo. I noticed that you have a [.DS](https://github.com/brainhack-school2020/emilyemchen-EEGML/blob/master/.DS_Store) file in your repository. Most likely that came over when committed all files in the repository and then [made a push from a Mac computer](https://stackoverflow.com/questions/107701/how-can-i-remove-ds-store-files-from-a-git-repository). If you want to keep files like these out of you repository, you can use a [.gitignore file](https://git-scm.com/docs/gitignore) that you add to the top of your github repository. This can also be helpful when you use a text editor or [IDE](https://vscode.readthedocs.io/en/latest/getstarted/settings/#creating-user-and-workspace-settings) that creates local configuration files that you don't want to upload to your repository.

**Comment by emilyemchen** [comment]
Hi @surchs! Thank you so much for this suggestion! I was wondering what the .DS file was. Would  the [.ipynb_checkpoints](https://github.com/brainhack-school2020/emilyemchen-EEGML/tree/master/.ipynb_checkpoints) directory also be something to include in the .gitignore file? Is that what you mean by a text editor/IDE creating local configuration files?

**Comment by surchs** [comment]
Yeah exactly. The ipynb checkpoints could also go in there. If you google around a bit you may find some nice `.gitignore` templates for python projects or those using notebooks.
